### PR TITLE
Vagrantfile: pre-built image instead of building

### PR DESCRIPTION
**Pull request by pookimon**
It will help us to save around 5 mins during pelux-manifest build

### CODE REVIEW

**Comment by sashko** [comment]
```
Setting up bitbake environment.
/vagrant/cookbook/yocto/initialize-bitbake.sh: line 36: /vagrant/pelux_yocto/sources/poky/oe-init-build-env: No such file or directory
```
Working directory must be `/home/yoctouser` now.

**Comment by martin-ejdestig** [comment]
This will mean updates to Dockerfile will not be reflected in the build. Or am I missing something?

I do not know what the history is here and I have no strong opinion one way or the other. Though I was under the impression that "updates to Dockerfile should be immediately reflected in builds" was something that was done by design.

**Comment by sashko** [comment]
> This will mean updates to Dockerfile will not be reflected in the build. Or am I missing something?
> 
> I do not know what the history is here and I have no strong opinion one way or the other. Though I was under the impression that "updates to Dockerfile should be immediately reflected in builds" was something that was done by design.

The idea is to have a separate job that updates image once Dockerfile is modified. See `OAI1209-567`

**Comment by thenor** [comment]
Commits should be signed off. @sashko could you help @pookimon with this?

**Comment by sashko** [comment]
> Commits should be signed off. @sashko could you help @pookimon with this?

Absolutely. I will make sure everything is done right. Right now it's WIP.

File: ci-scripts/do_repo_init [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # Copyright (C) 2017 Luxoft
 #
 
+set -x
```
Comment (sashko) [inline ci-scripts/do_repo_init]: remove all the changes from this file

Comment (pookimon) [reply ci-scripts/do_repo_init]: Done

File: ci-scripts/copy_to_archive [hunk 2]
```diff
@@ -4,7 +4,6 @@
 #
 
 
-set -x
```
Comment (sashko) [inline ci-scripts/copy_to_archive]: why is this still here?

File: ci-scripts/do_repo_init [hunk 3]
```diff
@@ -28,7 +27,6 @@ echo "yocto dir=${YOCTO_DIR}"
 
 # Copy the host repo to ensure that no later git commands destroy anything.
 COPY_DIR="/tmp/git_repo"
-whoami
```
Comment (sashko) [inline ci-scripts/do_repo_init]: and this

File: ci-scripts/do_repo_init [hunk 4]
```diff
@@ -3,7 +3,6 @@
 # Copyright (C) 2017 Luxoft
 #
 
-set -x
```
Comment (sashko) [inline ci-scripts/do_repo_init]: and this

**Comment by pookimon** [comment]
I can't see it in Files changed

**Pull request merged**